.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] [OPTIONAL] Kindly update the `linters.yml` file inside `.github/workflow` with the linters in [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) repo in order to run the `eslint` linters as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.). Now the eslint linters check are not running. 
![image](https://user-images.githubusercontent.com/24703171/215491025-1e9da9ac-84f9-4b40-9423-65810627997b.png)
This will mark as optional since it's not required by the previous reviewer but in an upcoming JS project, your PR will be marked as invalid.